.env

```diff
@@ -14,6 +14,10 @@ REDIS_PORT=6379
 SUPERVISOR_HOST=backend
```

Comment:
better naming: BACKEND_SUPERVISOR_HOST (and port). BUT: since this will break many deployers, we can leave this as is in .env. However within the code we need to use the backend prefix (see comments later), for better clarity.